test.js

```diff
@@ -0,0 +1,3 @@
+var test = function() {
```

Comment:
Unexpected var, use let or const instead   no-var<br>Expected a function declaration            func-style<br>'test' is assigned a value but never used  no-unused-vars<br>Missing space before function parentheses  space-before-function-paren